front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -148,29 +152,41 @@ const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
     project,
     phases
   );
+
   const isParticipationMethodIdeation = participation_method === 'ideation';
+
   const isParticipationMethodNativeSurvey =
-    participation_method === 'native_survey';
+    participation_method === 'native_survey' ||
+    currentPhase?.attributes.participation_method === 'native_survey';
```

Comment:
After our chat - refactor of this file will happen at a later time. Might be a bigger effort.